### PR TITLE
Leaderboard hit the API

### PR DESCRIPTION
**Pull request by Becky449**
##Created a new game with the name of your choice by using the API.
##Implemented the "Refresh" button that will get all scores for the game .
##Implement the form "Submit" button that will save a score for the game created by you (sending data to the API).
##Use arrow functions.
##Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+  body: JSON.stringify({
+    name: 'Handball tx',
+  }),
+}).then((res) => res.json())
+  .then((gameID) => {
+    const response = gameID.result;
+    gameIdentity = response.substring(14, response.lastIndexOf(' '));
+  });
```
Comment (HAMMAS-SALEEM) [inline src/index.js]: - [ ] You have implemented your api in a very great way 😊 but there is a problem. When we add something in your api through your form and refresh it and then api doesn't return us the old data which we added before. The problem is everytime your run your application it creates a new game id. A new game id means a new api. Kindly use only one game id even after refreshing the page 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Becky449 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/index.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Becky449  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**